analytics/lib/counts.py

```diff
@@ -293,7 +296,7 @@ def do_aggregate_to_summary_table(
 
 # called from zerver.actions; should not throw any errors
 def do_increment_logging_stat(
-    zerver_object: Union[Realm, UserProfile, Stream],
+    zulip_object: Union[Realm, UserProfile, Stream, RemoteZulipServer],
```

Comment:
`RemoteZulipServer` is from `zilencer` not `zerver`, so would be inaccurate to call this arg `zerver_objects` at this point.

Reply:
Cool, posted a proposed name change separately.